### PR TITLE
Fix typos; closes #429

### LINKED ISSUE #429: typos needing fixing
As flagged [here](https://github.com/solid/solid-ui/pull/427/files/064c7f29b95691315ee7d3eb98a2d12972f2131a..521b413659dd655c28c091c78b9a73b488a5b824#r703931226)

> Creative spellings --
> 
> * `defaultPreferedLangages`
> * `getPreferredLanagugesFor`
> 
> There may be others. To fix the above ---
> 
> * `s/Prefered/Preferred/g`
> * `s/Langages/Languages/g`
> * `s/Lanaguges/Languages/g`

@timbl did the [first global](https://github.com/solid/solid-ui/pull/427/commits/74d4c24f84e1d8bc53e48daea70b588339b1be9d), but not the other two.

And there is one more -- `s/deafults/defaults/g`

This will be much faster and easier for someone who works with command-line `git` and/or editors than for me, as I work entirely through the GitHub web interface.

### PR DESCRIPTION
**Pull request by janschill**
Regarding the talked about typos in https://github.com/solid/solid-ui/issues/429

### CODE REVIEW

**Review by TallTed — APPROVED** [review]
LGTM!

**Comment by bourgeoa** [comment]
Fail on lint

File: src/widgets/forms/autocomplete/language.ts [hunk 1]
```diff
@@ -24,20 +24,20 @@ export interface Binding {
 
 export const languageCodeURIBase = 'https://www.w3.org/ns/iana/language-code/' /// @@ unsupported on the web (2021)
 
-export const defaultPreferredLangages = ['en', 'fr', 'de', 'it', 'ar']
+export const defaultPreferredLanguages = ['en', 'fr', 'de', 'it', 'ar']
 
-export function addDefaults (array) {
+export function addDefaults(array) {
```
Comment (TallTed) [inline src/widgets/forms/autocomplete/language.ts:29]:
```suggestion
export function addDefaults (array) {
```

File: src/widgets/forms/autocomplete/language.ts [hunk 2]
```diff
@@ -24,20 +24,20 @@ export interface Binding {
...
   if (!array) array = []
-  return array.concat(defaultPreferredLangages.filter(code => !array.includes(code)))
+  return array.concat(defaultPreferredLanguages.filter(code => !array.includes(code)))
 }
 
-export async function getPreferredLanagugesFor (person: NamedNode) {
+export async function getPreferredLanguagesFor(person: NamedNode) {
```
Comment (TallTed) [inline src/widgets/forms/autocomplete/language.ts:34]:
```suggestion
export async function getPreferredLanguagesFor (person: NamedNode) {
```

File: src/widgets/forms/autocomplete/language.ts [hunk 3]
```diff
@@ -65,13 +65,13 @@ export async function getPreferredLanagugesFor (person: NamedNode) {
  *
  *  Either from solid preferences or browser preferences or default
  */
-export async function getPreferredLanguages () {
+export async function getPreferredLanguages() {
```
Comment (TallTed) [inline src/widgets/forms/autocomplete/language.ts:68]:
```suggestion
export async function getPreferredLanguages () {
```

File: src/widgets/forms/autocomplete/language.ts [hunk 4]
```diff
@@ -82,23 +82,23 @@ export async function getPreferredLanguages () {
...
+  return defaultPreferredLanguages
 }
 
 /* From an array of bindings with a names for each row,
  * remove dupliacte names for the same thing, leaving the user's
  * preferred language version
 */
 
-export function filterByLanguage (bindings, languagePrefs) {
+export function filterByLanguage(bindings, languagePrefs) {
```
Comment (TallTed) [inline src/widgets/forms/autocomplete/language.ts:93]:
```suggestion
export function filterByLanguage (bindings, languagePrefs) {
```

**Review by TallTed — CHANGES_REQUESTED** [review]
I don't know why these spaces were deleted in the first place ... but putting them back appears to be the path through the Lint error.

**Comment by janschill** [comment]
Should be good now, sorry about that. Is there a reason you have this enabled:

>First-time contributors need a maintainer to approve running workflows.

**Pull request merged**